### DISCUSSION TITLE
Make a prettify button

### DISCUSSION THREAD

**Discussion by guiIhermevieira** [apollographql/apollo-client-devtools#834] [Ideas]
Sometimes is actually hard to see those giant queries and their variables and actually understand what to remove and what to add if there's no prettify button

I think something along the lines of the prettify in GraphQL playground would work the best and improve for everyone

**Comment by cheapsteak** [comment]
Thanks for the feedback! There's a "Format Operation" button in the "..." menu, would that work? (there's also a hotkey for OPTION+SHIFT+F)

<img width="453" alt="image" src="https://user-images.githubusercontent.com/743976/159805001-5d0c7614-80ac-4b6b-b78b-6e308074e4ad.png">
